### PR TITLE
ch341a_spi.c: Change deprecated function

### PR DESCRIPTION
**Pull request by Asiderr**

### CODE REVIEW

File: ch341a_spi.c [hunk 1]
```diff
@@ -444,7 +444,11 @@ int ch341a_spi_init(void)
 		return -1;
 	}
 
-	libusb_set_debug(NULL, 3); // Enable information, warning and error messages (only).
+#if LIBUSB_API_VERSION >= 0x01000106
+   	libusb_set_option(NULL, LIBUSB_OPTION_LOG_LEVEL, LIBUSB_LOG_LEVEL_INFO);
```
Comment (macpijan) [inline ch341a_spi.c]: @Asiderr Have you confirmed that LOG_LEVEL_INFO corresponds to the old debug level of 3?

File: ch341a_spi.c [hunk 2]
```diff
@@ -444,7 +444,11 @@ int ch341a_spi_init(void)
...
+#else
+    libusb_set_debug(NULL, 3);
```
Comment (macpijan) [inline ch341a_spi.c]: @Asiderr Yout patch uses spaces not tabs for identation. Please take care of that.

**Review by macpijan — CHANGES_REQUESTED** [review]

File: ch341a_spi.c (hunk 1 above)
Comment (Asiderr) [reply ch341a_spi.c]: @macpijan Yes,  the LOG_LEVEL_INFO is equal 3. (http://libusb.sourceforge.net/api-1.0/group__libusb__lib.html lib is included in the file)

**Comment by Asiderr** [comment]
@macpijan Seems to be fixed.

**Review by macpijan — APPROVED** [review]

**Pull request merged**